tests/test_httpx.py

```diff
@@ -33,6 +34,21 @@
 )
 
 
+@pytest.mark.aiohttp
```

Comment:
I believe those two marks are not necessary?

Reply:
yes, I've realized it now, since the method doesn't use async anymore